mxabierto/adela
# Fixing unit test

### Changelog: Fixing 12 unit test and 3 pending unit test.

## Fixing 12 failed unit test
M: app/models/organization.rb
M: spec/features/admin/admin_manages_organizations_spec.rb
M: spec/features/user_uploads_organization_documents_spec.rb
M: spec/models/distribution_spec.rb
M: spec/models/organization_spec.rb
M: spec/requests/data_catalog_management_spec.rb
M: spec/requests/data_inventory_management_spec.rb

##Fixing 3 pending unit test.
M: spec/helpers/datasets_helper_spec.rb
M: spec/models/dataset_spec.rb

### How to test

* Following the next steps:
1) download this branch.
2) Startup your local environment.
3)  execute  bundle exec rspec spec in your console.
4) You must see the next result:

### Finished 
### 251 examples, 0 failures, 22 pending
